app/views/layouts/application.html.erb

```diff
@@ -18,7 +18,10 @@
 
   <div class="container">
     <div class="starter-template">
-
+      <% flash.each do |key, value| %>
+  			<%= content_tag :div, value, class: "flash #{key}" %>
```

Comment:
(Or, rather, make sure you don't have tabs, and make sure indenting is consistent with the rest of our views - which I *believe* is two spaces)